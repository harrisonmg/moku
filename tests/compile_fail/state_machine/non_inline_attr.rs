use moku::*;

#[state_machine]
mod blinky;

fn main() {}
