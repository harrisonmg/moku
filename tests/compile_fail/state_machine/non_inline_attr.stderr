error[E0658]: non-inline modules in proc macro input are unstable
 --> tests/compile_fail/state_machine/non_inline_attr.rs:4:1
  |
4 | mod blinky;
  | ^^^^^^^^^^^
  |
  = note: see issue #54727 <https://github.com/rust-lang/rust/issues/54727> for more information

error: a `moku::state_machine` module must be inline with its attribte, try
       ```
       #[moku::state_machine]
       mod blinky {
           ...
       }
       ```
 --> tests/compile_fail/state_machine/non_inline_attr.rs:4:1
  |
4 | mod blinky;
  | ^^^
